src/platform_linux/platform_inline.h

```diff
@@ -288,9 +289,20 @@ platform_log_stream_to_string(platform_stream_handle *stream)
       platform_log(Platform_default_log_handle, __VA_ARGS__);                  \
    } while (0)
 
-#define platform_error_log(...)                                                \
+/*
+ * For improved debugging-ability, prepend the user-supplied message string,
+ * 'msgfmt' with source-code location info.
+ */
+#define platform_error_log(msgfmt, ...)                                        \
    do {                                                                        \
-      platform_log(Platform_error_log_handle, __VA_ARGS__);                    \
+      platform_log(Platform_error_log_handle,                                  \
+                   "%s:%d:%s(): OS-pid=%d, OS-tid=%d, Thread-ID=%lu, " msgfmt, \
```

Comment:
Extra fields prepended to the error message ... as described in the issue and the commit message.